packages/components/src/components/hds/side-nav/index.ts

```diff
@@ -105,6 +127,13 @@ export default class HdsSideNav extends Component<HdsSideNavSignature> {
     return (this.isResponsive && !this.isDesktop) || this.isCollapsible;
   }
 
+  get ariaLabel(): string {
+    if (this.isMinimized) {
+      return this.args.ariaLabel ?? 'Open menu';
+    }
+    return this.args.ariaLabel ?? 'Close menu';
+  }
+
```

Comment:
at this point (since we don't need the old labels) it can even be
```suggestion
      return this.args.ariaLabel;
  }
```

Reply:
Updated this. thanks